rocAL/rocAL/source/augmentations/geometry_augmentations/node_crop.cpp

```diff
@@ -44,21 +44,55 @@ void CropNode::create_node()
 
     _crop_param->create_array(_graph);
 
-    _node = vxExtrppNode_CropPD(_graph->get(), _inputs[0]->handle(), _src_roi_width, _src_roi_height, _outputs[0]->handle(), _crop_param->cropw_arr,
-                                _crop_param->croph_arr, _crop_param->x1_arr, _crop_param->y1_arr, _batch_size);
+    int input_layout = (int)_inputs[0]->info().layout();
```

Comment:
Do we need to create a local variable? 
Can we not directly use this in API call for create scalar?